frontend/src/components/SettingsDialog.tsx

```diff
@@ -218,6 +236,27 @@ function SettingsDialog({ settings, setSettings }: Props) {
               </AccordionContent>
             </AccordionItem>
           </Accordion>
+
+          <Accordion type="single" collapsible className="w-full">
```

Comment:
Good for a first version. But from a UX perspective, I think this needs to be moved here at some point. I'll come up with a design for it.

<img width="357" alt="Screenshot 2024-06-10 at 1 10 07 PM" src="https://github.com/abi/screenshot-to-code/assets/23818/b1c2314e-0a52-45a4-aa6a-fe27c1a38ed0">

Reply:
Have attempted shifting it here if this is better